cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/cube/CubeDeleteQuery.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.metrics.store.cube;
+
+import co.cask.cdap.metrics.store.timeseries.MeasureType;
+import com.google.common.base.Joiner;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * Query that specifies delete entries from cube
+ */
+public class CubeDeleteQuery {
+
+  private final long startTs;
+  private final long endTs;
+  private final int limit;
```

Comment:
we don't use limit, do we? Does it make sense in case of delete? Maybe, if we use it. If not - remove